### DISCUSSION TITLE
Reinstatement Open For Almost 60 Days

### DISCUSSION THREAD

**Discussion by GoodbyeKittyy** [community/community#113684] [New to GitHub] [answered]
Hi everyone,

I made a reinstatement request previously and it is almost 60 days.. I can't transfer my repos too as when I tried to, there was a "Repository transfers not available for this account" message.

I have about 30 repos others need to assess.. If any support staff come across this, please refer to request #2618814 or #2638909

**Comment by queenofcorgis** [comment, marked as answer]
Hi @GoodbyeKittyy, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
